STANDARD.md

```diff
@@ -430,11 +430,32 @@ In addition, the following attribute is *recommended* (see the section
 on Unit Systems and Dimensionality, further below):
 
   - `gridUnitSI`
-    - type: *(float64 / REAL8)*
+    - type: 1-dimensional array containing N *(float64 / REAL8)*
```

Comment:
Note: This does not extend, but instead changes the definition of `gridUnitSI`. According to this wording, writing a single scalar to be used for all axes will no longer be supported for 2.0, meaning that all writers need to change this.
Do we additionally want to allow scalars here in 2.0, or should writers migrate?

Currently implementing this by printing a warning for legacy-defined scalar values for `gridUnitSI` in https://github.com/openPMD/openPMD-api/pull/1534/:
```cpp
Mesh &Mesh::setGridUnitSI(double gusi)
{
    setAttribute("gridUnitSI", gusi);
    if (auto standard = IOHandler()->m_standard;
        standard >= OpenpmdStandard::v_2_0_0)
    {
        std::cerr << "[Mesh::setGridUnitSI] Warning: Setting a scalar "
                     "`gridUnitSI` in a file with openPMD version '" +
                std::string(auxiliary::formatStandard(standard)) +
                "'. Consider specifying a vector instead in order to "
                "specify the gridUnitSI per axis (ref.: "
                "https://github.com/openPMD/openPMD-standard/pull/193).\n";
    }
    return *this;
}
```